src/lib/menu/menu-trigger.ts

```diff
@@ -407,7 +407,7 @@ export class MatMenuTrigger implements AfterContentInit, OnDestroy {
       positionStrategy: this._overlay.position()
           .flexibleConnectedTo(this._element)
           .withLockedPosition()
-          .withTransformOriginOn('.mat-menu-panel'),
+          .withTransformOriginOn('.mat-menu-panel, .mat-mdc-menu-panel'),
```

Comment:
Because the menu trigger is reused between the MDC and existing versions since it's just logic. I'm not using the same selector, because we discussed with @mmalerba that we should rename the selectors to avoid styles bleeding between components.